### ISSUE TITLE
Empathy Driven Development

### ISSUE THREAD

**Issue by ljgoslin** [a11yproject/a11yproject.com#611]
Accessibility is often forgotten by web developers, even today. [Join us on August 9th, from 1:0-2:00PM EST.](https://accessibility.deque.com/empathy-driven-development)

It’s not a surprise: without a focus on accessibility throughout the web development process, people with disabilities are often left behind. Fortunately, there are techniques and tools that can help kickstart the process. In this special live coding presentation from Deque Developer Advocate Marcy Sutton, you’ll learn hands-on skills for developing inclusively with JavaScript, HTML and CSS.

We’ll cover:

- Client-rendered application development with React.js
- Semantic HTML in JavaScript applications
- Focus management techniques and when to use them
- Disabling animations to keep your users from getting sick
- Writing automated tests to uphold quality

Accessibility doesn’t have to be boring or old-school! Cutting-edge web development techniques can be harnessed to benefit users of all abilities. By the end of this webinar, you should have a few technical tips and tricks to share with your colleagues and make the web a more inclusive place.

**Comment by svinkle** [comment]
Thanks! :)
